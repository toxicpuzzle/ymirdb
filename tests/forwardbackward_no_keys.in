FORWARD a
FORWARD b
BACKWARD a
BACKWARD b
bye