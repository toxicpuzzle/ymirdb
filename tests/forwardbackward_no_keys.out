> no such key

> no such key

> no such key

> no such key

> bye